framework/src/executioners/Executioner.C

```diff
@@ -203,7 +267,7 @@ Executioner::getTimeStepperName()
 bool
 Executioner::lastSolveConverged()
 {
-  return _fe_problem.converged();
+  return true;
```

Comment:
We alway converge?

Reply:
I possibly should make this pure virtual.

Reply:
Probably - aren't they overridden in our Executioners anyway?

Reply:
Yes. Now `FEProblemBase` does not own this flag. When it finish solving, it hands the flag to the caller. In `Transient`, the time stepper is holding the flag and in `Steady`, the executioner is holding. We could use `FEProblemBase::checkConvergence` here and then all the MOOSE executioners will no longer implement this. We can still make it virtual to give chance the derived executioners overriding it.

Reply:
Looks like time steppers have legislated reason to decide if the solve is converged or not. So I will just make this function pure virtual and force derived executioners to implement it.